prettier.py

```diff
@@ -58,6 +59,13 @@ def prettify_code(edit, view, region):
 # Run prettier an a file
 class PrettierCommand(sublime_plugin.TextCommand):
     def run(self, edit):
+        extensions = settings.get('extensions')
+
+        # Check if the file extension is allowed
+        file_extension = os.path.splitext(self.view.file_name())[1][1:]
```

Comment:
This code is inspired by this plugin: https://github.com/TheSavior/ESLint-Formatter/blob/master/ESLint-Formatter.py#L125